CakePHP/Sniffs/ControlStructures/ElseIfDeclarationSniff.php

```diff
@@ -15,34 +15,39 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
+namespace CakePHP\Sniffs\ControlStructures;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
 /**
  * Ensures that elseif is used instead of else if
  *
  */
-class CakePHP_Sniffs_ControlStructures_ElseIfDeclarationSniff implements PHP_CodeSniffer_Sniff
+class ElseIfDeclarationSniff implements Sniff
 {
 
-/**
- * Returns an array of tokens this test wants to listen for.
- *
- * @return array
- */
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
```

Comment:
I prefer `@inheritdoc` for external API doc blocks if they have a good doc block already.